src/EnlumineurFormatter/EFContext.class.st

```diff
@@ -287,6 +288,17 @@ EFContext >> numberOfNewLinesAfterMethodSignature: aBoolean [
 	newLinesAfterMethodSignature := aBoolean
 ]
 
+{ #category : #accessing }
+EFContext >> numberOfNewLinesAfterMethodSignatureWithMethodComment [
+
+	^ newLinesAfterMethodSignatureWithMethodComment
+]
+
+{ #category : #accessing }
+EFContext >> numberOfNewLinesAfterMethodSignatureWithMethodComment: aBoolean [
+	newLinesAfterMethodSignatureWithMethodComment := aBoolean
```

Comment:
```suggestion
EFContext >> numberOfNewLinesAfterMethodSignatureWithMethodComment: anInteger [
	newLinesAfterMethodSignatureWithMethodComment := anInteger
```

Reply:
But then this is already wrong in the setting that was there before (newLinesAfterMethodSignature).

I will change it

Reply:
@gcotelli fixed in https://github.com/pharo-project/pharo/pull/12178/commits/00831674cee35ea901bb14f75fd17f03c83a2fda